kythe/typescript/indexer.ts

```diff
@@ -854,7 +854,9 @@ class Visitor {
     // This can happen in cases of importing local modules, like
     //   declare namespace Foo {}
     //   import foo = Foo;
-    if (ts.isSourceFile(sf)) {
+    // Check that the value declaration actually exists. TypeScript doesn't
```

Comment:
Maybe include a reference to the TypeScript issue as a TODO here, so that it can be backed out when that issue is resolved.

Reply:
Done.